.github/scripts/makeLambdaFunction.yml

```diff
@@ -0,0 +1,23 @@
+Parameters: 
+  LambdaName: 
+    Description: Unique name of your Lambda function.
+    Type: String
+    
+Resources:
+  makeLambdaFunction:
+    Type: AWS::Lambda::Function
+    Properties:
+      FunctionName: !Ref LambdaName
+      Role: arn:aws:iam::061431082068:role/Lambda_Role
+      Runtime: nodejs14.x
+      Handler: index.handler
+      Code:
+        ZipFile: |
```

Comment:
I noticed that this is actually the old version of the makeLambdaFunction.yml script, you'll want to make sure to use the more recent version of the makeLambdaFunction.yml script which is in the devops/makeLambdaFunction folder in the development branch, as that contains the routing will zips the code from the data/simpleLambda folder